tuxemon/core/components/event/actions/npc.py

```diff
@@ -70,20 +70,18 @@ def create_npc(self, game, action):
             return
 
         # Get the npc's parameters from the action
-        name = str(action.parameters[0])
-        tile_pos_x = int(action.parameters[1])
-        tile_pos_y = int(action.parameters[2])
-        animations = str(action.parameters[3])
-        behavior = str(action.parameters[4])
+        slug = str(action.parameters[0])
+        tile_pos_x = int(action.parameters[2])
+        tile_pos_y = int(action.parameters[3])
+        animations = str(action.parameters[4])
+        behavior = str(action.parameters[5])
 
         # Ensure that the NPC doesn't already exist on the map.
-        event_engine = game.event_engine
-        npc_exists = Condition("npc_exists", [name], 1, 1, "is", 0, 0)
-        if event_engine.conditions["npc_exists"]["method"](game, npc_exists):
+        if slug not in world.npcs:
```

Comment:
This should be: `if slug in world.npcs:`

This code was originally there to prevent map makers from accidentally adding the same NPC over and over again if they improperly created a `create_npc` event that didn't check to see if the NPC already existed.